econml/tests/test_drlearner.py

```diff
@@ -793,9 +793,12 @@ def test_groups(self):
             est.fit(y, t, W=w, groups=groups)
 
         # test outer grouping
-        # NOTE: we should ideally use a stratified split with grouping, but sklearn doesn't have one yet
+        # NOTE: StratifiedGroupKFold has a bug when shuffle is True where it doesn't always stratify properly
```

Comment:
Is this bug worth worrying about for our users since we use crossfit uses StratifiedGroupKFold with shuffle=True?

Reply:
Hopefully it will be fixed in sklearn and then it will have the right behavior, but until then it's possible that users can run into it (although the buggy behavior only occurs with certain datasets, so hopefully it works most of the time).

However I don't think there's any good fix on our end - in general we do want to shuffle, it's just that for the purposes of this one test we can ignore that, but it wouldn't be an appropriate substitute in general.